### PR TITLE
get rid of MagicFolderCLITestMixin in CreateMagicFolder tests

### PR DESCRIPTION
**Pull request by vu3rdd**
Initial draft of work towards #22, specifically to remove the dependency of no_network module.

### CODE REVIEW

File: src/magic_folder/test/cli/test_magic_folder.py [hunk 1]
```diff
@@ -589,143 +588,366 @@ def addOutcomeDetails(testcase, outcome):
...
-            self.collective_dircap, self.upload_dircap = self.get_caps_from_files(0)
-        d.addCallback(get_caps)
-        d.addCallback(lambda ign: self.check_joined_config(0, self.upload_dircap))
-        d.addCallback(lambda ign: self.check_config(0, abs_local_dir_u))
-        return d
+        self.tempdir = self.client_fixture.tempdir
+        self.node_directory = self.client_fixture.node_directory
+
+    @defer.inlineCallbacks
+    def test_create_magic_folder(self):
```
Comment (exarkun) [inline src/magic_folder/test/cli/test_magic_folder.py]: A docstring for this and the other test methods would be great.

File: src/magic_folder/test/cli/test_magic_folder.py [hunk 2]
```diff
@@ -589,143 +588,366 @@ def addOutcomeDetails(testcase, outcome):
...
+        # Get a magic folder.
+        magic_folder = self.tempdir.child(u"magic-folder")
+        outcome = yield cli(
+            self.node_directory, [
+            b"create",
+            b"magik:",
```
Comment (meejah) [inline src/magic_folder/test/cli/test_magic_folder.py]: personally, i would indent these (in the list) another level so it's more obvious they're "parts of a single arg"  .. but maybe that's just a style thing :)

Comment (exarkun) [reply src/magic_folder/test/cli/test_magic_folder.py]: I could get behind running "black" on the whole codebase and the enforcing it all in CI, I think.

Comment (vu3rdd) [reply src/magic_folder/test/cli/test_magic_folder.py]: Fixed the formatting and added docstrings in the commits 0f316de and 
9c4ac61. Thanks.

**Comment by vu3rdd** [comment]
Fixed the formatting and added docstrings in the commits 0f316de and
9c4ac61. Thanks.

**Comment by vu3rdd** [comment]
closes #47

**Pull request merged**